hooks/pre-commit

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
this file won't be applied automatically ( hooks stored in `.git/hooks` so it's our of git repo) - use `husky` package and `lint-staged` for running on staged files